tests/cloudformation/checks/resource/aws/example_SecurityGroupRuleDescription/SecurityGroupRuleDescription-FAILED.yaml

```diff
@@ -1,21 +1,10 @@
-Description: Security Group Example
```

Comment:
@twmartin This is a good test for variable rendering, why remove the parameters block?

Reply:
I feel as though it is beneficial to keep unit tests stripped as minimally as reasonably possible while still providing adequate coverage to the test case. Perhaps variable rendering test coverage should live in its own explicit test and not passively in the `SecurityGroupRuleDescription` unit test. With the absence of the parameters block from this file, is test coverage for variable rendering completely lost, or is it still retained by way of other example cfn templates/tests?